src/Chapter2.hs

```diff
@@ -621,7 +623,9 @@ Write a function that takes elements of a list only on even positions.
 >>> takeEven [2, 1, 3, 5, 4]
 [2,3,4]
 -}
-takeEven = error "takeEven: Not implemented!"
+takeEven :: [Int] -> [Int]
+takeEven (x:y:zs) = x : takeEven zs
```

Comment:
As you don't use `y` in the right part, you can use a wildcard for the pattern matching:
```suggestion
takeEven (x:_:zs) = x : takeEven zs
```